packages/web/src/builders/package/package.impl.ts

```diff
@@ -177,10 +175,15 @@ export function run(
               ),
             { success: true }
           ),
+          last(),
           tap({
-            complete: () => {
-              updatePackageJson(options, context, target, dependencies);
-              context.logger.info('Bundle complete.');
```

Comment:
previously, if compilation (or anything else) failed, we'd still generate a `dist` folder with a `package.json`